runtimes/bundles/org.jboss.tools.rsp.server.minishift/src/main/java/org/jboss/tools/rsp/server/minishift/discovery/MinishiftBeanType.java

```diff
@@ -75,10 +75,9 @@ private MinishiftVersions findOrLoad(File root) {
 	@Override
 	public String getUnderlyingTypeId(File root) {
 		MinishiftVersions props = findOrLoad(root);
-		if( props != null ) {
-			if( props.getCDKVersion() != null ) {
-				return "CDK";
-			}
+		if( props != null 
+				&& props.getCDKVersion() != null ) {
```

Comment:
couldn't this be on one line?  if( props != null && props.getCDKVersion() != null ) {

Reply:
I prefer to separate both condition to the max and thus having every condition on a separate line